agito.py

```diff
@@ -39,6 +39,10 @@
 MERGEINFO_LINE_RE = re.compile(r'([\w/\-]+):.*-(\d+)')
 WORD_MATCH = re.compile(r'(\s*)(\S+)')
 BULLET_POINT_RE = re.compile(r'(\s*[\*\+]\s*)')
+COMMIT_REF_REs = (
```

Comment:
It's still a constant, and convention for constants is all-uppercase. Can you just make it `COMMIT_REF_RES`? Sorry for nitpick.